packages/mgt-components/src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -87,7 +87,9 @@ export class MgtPersonCard extends MgtTemplatedComponent {
    * @return {*}  {string[]}
    * @memberof MgtPersonCard
    */
-  public static getScopes(): string[] {
+  public static requiredScopes: string[] = MgtPersonCard.getScopes();
+
+  private static getScopes(): string[] {
```

Comment:
we should keep `getScopes` public but deprecated, to avoid breaking changes

Reply:
I thought you mentioned that it hasn't been released yet, which is why I changed it to `private`.